tests/other/no-exceptions/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#===============================================================================
+# Copyright 2016-2018 Intel Corporation
```

Comment:
`2016-2018` -> `2019`

Reply:
`tests/other/no-exceptions/CMakeLists.txt`

Minor, but we mostly use underscore (not hyphen) as a separator for file/directory names inside the repo. Could you please rename it to `no_exceptions`?